Makefile

```diff
@@ -28,7 +28,9 @@ endif
 
 build:
 	mkdir -p ${PKGDIR}
-	gox -output="${PKGDIR}/{{.Dir}}_${OS}_${ARCH}" -os="${OS}" -arch="${ARCH}"
+	for x in "$(OS_LIST)" ; do \
+		gox -output="${PKGDIR}/{{.Dir}}_{{.OS}}_{{.Arch}}" -os="$$x" -arch="${ARCH}" ; \
+	done
```

Comment:
@akm 

```for x in "$(OS_LIST)" ; do \``` expand to  ```for x in "linux darwin" ; do \```
therefore always loops one time.

gox's -os parameter accepts multiple values saparated by space.

> gox --help
>
> -os=""              Space-separated list of operating systems to build for

so, this code will walk well, but meaningless "for".